usr/bin/tce-update

```diff
@@ -90,11 +88,7 @@ check_dependencies(){
 				REPO=`grep -w "$TARGET" md5.db`
 				if [ -n "$REPO" ]; then
 					echo -n  "Fetching required dependency: $TARGET "
-					if [ -n "$USEZSYNC" ]; then
-						fetchzsync "$TARGET" 2>/dev/null
-					else
-						fetch "$TARGET" 2>/dev/null
-					fi
+					fetchzsync "$TARGET" 2>/dev/null
```

Comment:
Same as previous.  I get that you are trying to save a couple of lines of code around if statements, but Its not intuitive to hide the check.

Reply:
ok. changed back